INTITA/Modules/LogInScreen/LogInViewController.swift

```diff
@@ -103,18 +104,20 @@ class LogInViewController: UIViewController, Storyboarded, AlertAcceptable {
 extension LogInViewController: UITableViewDelegate, UITableViewDataSource {
     
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        return 7
+        return 8
     }
     
     func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
-        
+
         let nameCell = LoginCells(rawValue: indexPath.row)
         var heightCell: CGFloat
-        
+
         switch nameCell {
+        case .emptyCellOne:
+            heightCell = 45
         case .logoImageCell:
             heightCell = 243
-        case .emptyCellOne:
+        case .emptyCellTwo:
             heightCell = 52
```

Comment:
heightCell means cell that contains height. Please replace it with correct names. Such as cellHeight or cellWidth.

Reply:
It was renamed.